hypertrace-graphql-gateway-service-metric-utils/src/main/java/org/hypertrace/graphql/utils/metrics/gateway/MetricContainerMapConverter.java

```diff
@@ -71,13 +70,13 @@
         ConvertedMetricContainer::new);
   }
 
-  private static class ConvertedMetricContainer extends ConvertedAggregationContainer
+  private static class ConvertedMetricContainer extends BaselineConvertedAggregationContainer
```

Comment:
make sure implementation and interface names stay in sync

Reply:
done